### PR TITLE
trial

### PR DESCRIPTION
**Pull request by amalss18**
trial

### CODE REVIEW

File: pdfToText.py [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
-	pageObj = pdfReader.getPage(i) 	
+	pageObj = pdfReader.getPage(i)
 	text = pageObj.extractText()
 	nxtPage = '\nThis is page ' + str(i+1) + '\n\n'
 	text+=nxtPage
 	file.write(text)
 file.close()
-pdfFileObj.close() 
+pdfFileObj.close()
+#Trial
```
Comment (harshbaldwa) [inline pdfToText.py]: bhai pagal hai kya

File: pdfToText.py [hunk 2]
```diff
@@ -1,15 +1,16 @@
-import PyPDF2 
+import PyPDF2
 
 
 file = open('text.txt', 'w')
 path = input("Enter the file path : ").strip()
-pdfFileObj = open(path, 'rb') 
-pdfReader = PyPDF2.PdfFileReader(pdfFileObj) 
+pdfFileObj = open(path, 'rb')
+pdfReader = PyPDF2.PdfFileReader(pdfFileObj)
```
Comment (harshbaldwa) [inline pdfToText.py]: kyun bhai kyun

Comment (amalss18) [reply pdfToText.py]: trial

Comment (harshbaldwa) [reply pdfToText.py]: i am happy

**Pull request merged**